lib/Modules/OVSCharmUtils/OVSCharmUtils.psm1

```diff
@@ -281,47 +293,66 @@ function Disable-OVS {
             $ovsRunning = $false
         }
     }
-
     if($ovsRunning) {
         $bridges = Start-ExternalCommand { & $OVS_VSCTL list-br }
         foreach($bridge in $bridges) {
             Start-ExternalCommand { & $OVS_VSCTL del-br $bridge }
         }
     }
-
     foreach($svcName in $ovsServices) {
         $svc = Get-Service -Name $svcName -ErrorAction SilentlyContinue
         if($svc) {
             Stop-Service $svcName -Force
             Disable-Service $svcName
         }
     }
-
     Disable-OVSExtension
 }
 
 function Enable-OVS {
     Enable-OVSExtension
-
     $ovsServices = @($OVS_OVSDB_SERVICE_NAME, $OVS_VSWITCHD_SERVICE_NAME)
     foreach($svcName in $ovsServices) {
         Enable-Service $svcName
         Start-Service $svcName
     }
+    Invoke-JujuCommand -Command @($OVS_VSCTL, 'set-manager', 'ptcp:6640:127.0.0.1')
```

Comment:
It is the default, but we had it changed until now, because the `native` interface was not yet supported on Windows. Now, we're good.